src/Sulu/Bundle/ResourceBundle/Resources/public/js/components/filters/util/toolbar-extension-handler.js

```diff
@@ -28,7 +28,7 @@ define(['app-config'], function(AppConfig) {
          * @param filterResultSelector {String}
          */
         extendToolbar = function(context, toolbarItems, toolbarInstanceName, dataGridInstanceName, filterResultSelector) {
-            if (!!context && !!toolbarItems && !!dataGridInstanceName) {
+            if (!!context && !!toolbarItems && !!dataGridInstanceName && !!this.config.contexts[context]) {
```

Comment:
A context is something like `contacts` or `products` and is used to decide which filters belong to which list and when to display the filter stuff